src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -2272,9 +2272,105 @@ def reindex(
         else:
             return self._reindex_axis_1(labels=labels, **kwargs)
 
+    def is_monotonic_decreasing(self, index: bool) -> "SnowflakeQueryCompiler":
+        """
+        Returns a QueryCompiler containing only a column that checks for monotonically
+        decreasing values.
+
+        Parameters
+        ----------
+        index: bool
```

Comment:
If you implement the index version of this method as `index.to_series().is_monotonic_increasing/decreasing()` you wouldn't need the `index` parameter here and can shorten the logic here too. Most index methods convert the index to series and then call the series version of the method.

Reply:
Done!